ubuntu-advantage

```diff
@@ -80,6 +80,8 @@ Commands:
  enable-livepatch <TOKEN>          enable the Livepatch service
  disable-livepatch [-r]            disable the Livepatch service. With "-r", the
                                    canonical-livepatch snap will also be removed
+ install-cc <TOKEN>                enable the commoncriteria PPA repository and
+                                   install the ubuntu-commoncriteria .DEB package
```

Comment:
Ok, I thought a bit more about this, and what do you think about calling this new command `enable-cc-provisioning`. I think it represents better what is happening, and will allow us to use all the service-related functions we already have in the code.

Reply:
I'd have to agree. That makes more sense. Should the service be called cc-provisioning then?

Reply:
yes

Reply:
Done. Changed the flag to "cc_provisioning" to keep with the bash variable naming convention. The feature is tested and working. I see some assertion failures on the updated python tests with new command (also committed in this patch). I will check them again today.